test/plots/linePlotTests.ts

```diff
@@ -683,4 +683,157 @@ describe("Plots", () => {
 
     });
   });
+
+  describe("Line Plot", () => {
```

Comment:
I took this from the previous iteration of this pull request.  I can change it back.